airbyte-scheduler/scheduler-persistence/src/main/java/io/airbyte/scheduler/persistence/JobNotifier.java

```diff
@@ -101,28 +101,29 @@ private void notifyJob(final String reason,
               workspaceId,
               action,
               MoreMaps.merge(jobMetadata, sourceMetadata, destinationMetadata, notificationMetadata.build()));
-          switch (action) {
-            case FAILURE_NOTIFICATION:
-              if (!notificationClient.notifyJobFailure(sourceConnector, destinationConnector, jobDescription, logUrl)) {
-                LOGGER.warn("Failed to successfully notify failure: {}", notification);
-              }
-              break;
-            case SUCCESS_NOTIFICATION:
-              if (!notificationClient.notifyJobSuccess(sourceConnector, destinationConnector, jobDescription, logUrl)) {
-                LOGGER.warn("Failed to successfully notify success: {}", notification);
-              }
-              break;
-            case CONNECTION_DISABLED_NOTIFICATION:
-              if (!notificationClient.notifyConnectionDisabled(workspace.getEmail(), sourceConnector, destinationConnector, jobDescription,
-                  workspaceId, connectionId)) {
-                LOGGER.warn("Failed to successfully notify auto-disable connection: {}", notification);
-              }
-              break;
-            case CONNECTION_DISABLED_WARNING_NOTIFICATION:
-              if (!notificationClient.notifyConnectionDisableWarning(workspace.getEmail(), sourceConnector, destinationConnector, jobDescription,
-                  workspaceId, connectionId)) {
-                LOGGER.warn("Failed to successfully notify auto-disable connection warning: {}", notification);
-              }
+
+          if (FAILURE_NOTIFICATION == action) {
```

Comment:
looking at this again, thinking maybe I should use a default and log something like "unknown action"?